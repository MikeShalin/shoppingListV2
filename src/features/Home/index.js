/** @flow **/
import * as React from 'react'
import { Segment } from 'semantic-ui-react'

import ProductForm from 'Features/ProductForm'
import ProductList from 'Features/ProductList'
import Wrapper from 'Core/styled/Wrapper'

const FormExampleForm = () => (
  <Wrapper>
    <Segment>
      <ProductForm/>
      <ProductList/>
    </Segment>
  </Wrapper>
)

export default FormExampleForm