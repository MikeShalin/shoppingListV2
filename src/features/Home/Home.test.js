import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Home from './index'

describe('Home component', () => {
  it('matches early snapshots', () => {
    const tree = shallow(<Home />)
    expect(shallowToJson(tree)).toMatchSnapshot()
  })
})