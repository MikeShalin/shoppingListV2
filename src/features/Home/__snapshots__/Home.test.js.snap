// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches early snapshots 1`] = `
<Segment>
  <inject-withHandlers(FormExampleForm)-with-productForm />
  <inject-ListExampleDivided-with-productList />
</Segment>
`;

exports[`Home component matches early snapshots 1`] = `
<Segment>
  <inject-withHandlers(FormExampleForm)-with-productForm />
  <inject-ListExampleDivided-with-productList />
</Segment>
`;
